delegates.txt

```diff
@@ -72,6 +72,7 @@ Claude Pache (CPE)
 Conrad Watt (CWT)
 Conrad Watts (CWS)
 Corey Frang (CF)
+Cuili (CLI)
```

Comment:
who's this?

Reply:
https://github.com/tc39/Admin-and-Business/issues/133

Reply:
ah, gotcha. what's their full name?